app/controllers/comments_controller.rb

```diff
@@ -10,12 +10,13 @@ def create
     comment.user = current_user
     comment.save
 
-    respond_with(comment, location: article_path(article))
+    render partial: "comments/comment"
```

Comment:
i think it's better to use = render comment even if decorators used

to git this working we need decorator to support to_partial method, right?